Packs/SpyCloudEnterpriseProtection/Scripts/SpyCloudEnterpriseFeed/SpyCloudEnterpriseFeed.yml

```diff
@@ -0,0 +1,82 @@
+comment: This Script execute spycloud-watchlist-command and create incident objects.
+commonfields:
+  id: SpycloudWatchlistData
+  version: -1
+name: SpycloudWatchlistData
```

Comment:
I see the script's name is different than the file name, can we change the file name to match? if yes can we change the FeedLastRun name as well? It might be better to refrain from using the name Feed as we have a feed mechanism.